src/classes/CW_AssetTypeApprovalProcess.cls

```diff
@@ -0,0 +1,34 @@
+/**
+* File Name   :    CW_AssetTypeApprovalProcess.cls
+* Description :    B-108927 Approve / Reject Mailer Dispatch Dates
+*				   B-108925 Approve / Reject Teaser Dates
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 16/05/2014 Cristina P.	Intial
+*
+*/
+public with sharing class CW_AssetTypeApprovalProcess 
+{
+	public void submitAssetTypeForApproval(Id messagingId)
+	{
+		CW_Messaging__c msg = [SELECT Dispatch_Date__c,Live_Date__c, End_Date__c
```

Comment:
Problem: Unbound SOQL query. Reason: a SOQL query can return up to 50,000 records. If this number is exceeded a runtime exception is thrown. Suggestion: cap the max number of records returned adding a LIMIT clause to the query.